src/java.base/share/classes/java/lang/foreign/SymbolLookup.java

```diff
@@ -130,6 +130,17 @@ public interface SymbolLookup {
      */
     Optional<MemorySegment> find(String name);
 
+    /**
+     * Returns the address of the symbol with the given name.
+     * @implSpec the default implementation for this method calls {@code this.find(name).get()}.
+     * @param name the symbol name.
+     * @return a zero-length memory segment whose address indicates the address of the symbol.
+     * @throws java.util.NoSuchElementException if no symbol with the given name exists.
+     */
+    default MemorySegment get(String name) {
+        return find(name).get();
+    }
```

Comment:
Maybe provide a better error message:
```suggestion
    /**
     * Returns the address of the symbol with the given name.
     * @implSpec the default implementation for this method calls {@code this.find(name).orElseThrow()}.
     * @param name the symbol name.
     * @return a zero-length memory segment whose address indicates the address of the symbol.
     * @throws java.util.NoSuchElementException if no symbol with the given name exists.
     */
    default MemorySegment get(String name) {
        var address = this.find(name);

        // Avoid using a capturing lambda
        if (address.isPresent()) {
            return address.orElseThrow();
        }
        throw new NoSuchElementException("Unable to find a native symbol named '" + name + "'");
    }
```

See also: https://github.com/openjdk/jdk/pull/15286

https://github.com/openjdk/panama-foreign/blob/c4e05f5c25910e5e23d3f6af585d066f671fb9b7/src/java.base/share/classes/jdk/internal/foreign/support/DefaultNativeLookupUtil.java#L47-L57